doc/src/manual/parallel-computing.md

```diff
@@ -1438,6 +1438,41 @@ julia> a
 
 Note that [`Threads.@threads`](@ref) does not have an optional reduction parameter like [`@parallel`](@ref).
 
+Julia supports accessing and modifying values *atomically*, that is, in a thread-safe way to avoid
+[race conditions](https://en.wikipedia.org/wiki/Race_condition). A value (which must be of a primitive
```

Comment:
Actually not all `primitive types` are supported. 
The lists is: `Int8, Int16, Int32, Int64, Int128, UInt8, UInt16, UInt32, UInt64, UInt128, Float16, Float32, Float128`. With the exception that `Int128` and `UInt128` don't work on AArch32 (and last I checked ppc64le).

Reply:
This doesn't claim that all primitive types have atomic operations, just that being a primitive type is a necessary requirement. Perhaps a footnote itemizing these types as in this comment?